### ISSUE TITLE
tutorial?

### ISSUE THREAD

**Issue by njmii** [vikingmicrowave/Microwave-Open-Source#9]
i am new in this area, can someone spare their time making a video tutorial? it would be greatly appreciated

**Comment by bumbojumbo** [comment]
I would also be interested in a video.

**Comment by sonicrules11** [comment]
Tomorrow I'll try to explain what to do for you guys

**Comment by bumbojumbo** [comment]
Okay, a few screenshots would be fantastic. Thanks so much.

**Comment by trinialol** [comment]
I'm gonna make a video for you guys.

**Comment by bumbojumbo** [comment]
Thanks a ton!

**Comment by njmii** [comment]
thank you everyone

**Comment by trinialol** [comment]
Will upload a video later, when I'm back at home.

**Comment by dentmaged** [comment]
Thanks @trinialol! I'm interested in seeing it too.

**Comment by trinialol** [comment]
https://jerr0w.xyz/dl/microwave-tutorial.mp4

PS: You have to create your own "inject"-File.

You can create it by writing "nano inject" and just enter the command + parameters and press CTRL+X and confirm to save with Y.

**Comment by bumbojumbo** [comment]
are those files at 1:26 necessary to be inside the main folder?

**Comment by trinialol** [comment]
Yes. These files are necessary.

**Comment by bumbojumbo** [comment]
can you upload your folder with everything already in it?

**Comment by bumbojumbo** [comment]
compress then upload to google drive?

**Comment by trinialol** [comment]
https://jerr0w.xyz/dl/Microwave-Open-Source.zip

**Comment by bumbojumbo** [comment]
thanks

**Comment by trinialol** [comment]
You're welcome.

**Comment by bumbojumbo** [comment]
thanks it works.

**Comment by bumbojumbo** [comment]
what is the reason behind the inject file is the sudo ./osxinj csgo_osx64 libMicrowave.dylib not as good a method?

**Comment by trinialol** [comment]
The reason behind the inject file is, that you don't have to write that much anymore. just do "sudo ./inject" and thats it.

**Comment by bumbojumbo** [comment]
Oh thanks didn't realize it was that easy, on unknown cheats someone made a
skinchanger with it you can check it out if you want, it's on page 3 of
microwave forums

**Comment by dentmaged** [comment]
Can you upload it to YouTube? School blocks any unknown servers and it takes ages to unblock.

**Comment by trinialol** [comment]
Watch it at home then? wtf

**Comment by trinialol** [comment]
oh and i know about the skinchanger. I got that version too. That skinchanger exists because of me. I debugged it for the coder.

**Comment by bumbojumbo** [comment]
Oh didn't realize, do get it, would I need to recompile the injector files
or just the libmicrowave, because I tried compiling the libmicrowave but it
gave 4 errors.

**Comment by dentmaged** [comment]
@trinialol I'm at a boarding school. I'm there all the time.

**Comment by bumbojumbo** [comment]
He sent the pre compiled files, just do path to his microwave open source,
sudo ./inject

**Comment by bumbojumbo** [comment]
https://jerr0w.xyz/dl/Microwave-Open-Source.zip if you didn't see it.

**Comment by trinialol** [comment]
Exactly... if you can't download it from my server, then I can't help you. It's not really big. Just download it through your Smartphone Hotspot or something.

**Comment by bumbojumbo** [comment]
If your school blocks google drive I don't know what else to suggest.
https://drive.google.com/open?id=0B1te9AQrixqXWWxsdmF2WTBGSVk

**Comment by dentmaged** [comment]
@bumbojumbo I was asking the video, but don't worry. It's fine.

**Comment by ANDREW30042004** [comment]
Does this cheat has skin changer or aimbot?

**Comment by bumbojumbo** [comment]
Join the discord.

**Comment by KuruKuru3532** [comment]
can you please reupload the video

**Comment by KuruKuru3532** [comment]
the link doesn't work anymore

**Comment by adamsager04** [comment]
replied das video pelz

**Comment by adamsager04** [comment]
replied das video pelz
